about.html

```diff
@@ -0,0 +1,146 @@
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <main>
+      <section id="About" class="home_section">
+        <div class="home_section_title">
+          <h2>"Hello! Sharing world"</h2>
+          <h1>NEED TO VISIT RWANDA? JOIN THE GORILLA NAMING CEREMONY.</h1>
+        </div>
+        <article>
+          <h4>
+            Kwita Izina is inspired by the ancient Rwandan Tradition of naming a
+            baby soon after its birth. During this ceremony, infant mountain
+            gorillas that were born in Rwanda during the previous year receive
+            their names as well as one newly formed family.
+          </h4>
+        </article>
+        <div class="info">
+          <p>
+            Please contact us per Email for more informations about the gorilla
+            naming ceremony.
+          </p>
+          <div class="link">
+            <a href="#">kwitaizina@volcanoes.rw</a>
+          </div>
+        </div>
+      </section>
+      <section id="Program" class="overView d-flex flex-center">
+        <h2>Gorilla Naming Ceremony Overview</h2>
+        <p>
+          Kwita Izina is a Ceremony that happens every year, and Hundreds of
+          people from different locations are welcomed in Rwanda for that
+          ceremony.
+        </p>
+        <div class="logo">
+          <img src="./images/main-logo.png" alt="" />
+        </div>
+        <div class="line"></div>
+      </section>
+
+      <section id="Team" class="gorilla">
+        <h2>Main Ceremony's Activities.</h2>
+        <p>
+          Welcome to the Gorilla Naming Ceremony! Join us to become a part of us
+          in forming gorilla families by giving name to the born babies. You
+          will enjoy hiking and playing with gorilla's babies.
+        </p>
+        <ul class="image-list d-flex flex-center">
+          <li>
+            <img src="images/picture1.jpg" alt="Gorilla image" />
+            <p class="text">
+              2020 <br />
+              Volcanoes Hiking
+            </p>
+            <span class="backgrd"></span>
+          </li>
+          <li>
+            <img src="images/picture2.jpg" alt="Gollira image" />
+            <p class="text">
+              2021 <br />
+              Gorilla Families
+            </p>
+            <span class="backgrd"></span>
+          </li>
+        </ul>
+      </section>
+
+      <section id="Partner" class="partners_section">
+        <h2 class="partners-title">Our Partners</h2>
+        <ul class="partner-list d-flex flex-center">
+          <li><img src="./images/partners/1.png" alt="partner image1" /></li>
+          <li><img src="./images/partners/2.png" alt="partner image2" /></li>
+          <li><img src="./images/partners/3.png" alt="partner image3" /></li>
+          <li><img src="./images/partners/4.png" alt="partner image4" /></li>
+          <li><img src="./images/partners/5.png" alt="partner image5" /></li>
+        </ul>
```

Comment:
- [ ] Good job working on this page but please kindly remove it as it is not specified in the design

![image](https://user-images.githubusercontent.com/43995830/164577280-68bbaec0-77f0-470b-a45c-21e3a7eb7074.png)

Reply:
resolved 👍 Kindly add the partner's section back to the home page as it is supposed to be present there. It should be removed only in the about page.